Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -13,7 +13,11 @@ class AppSettingsTableViewController: SettingsTableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        navigationItem.title = .AppSettingsTitle
+        let variables = Experiments.shared.getVariables(featureId: .nimbusValidation, sendExposureEvent: true)
+        let title = variables.getText("settings-title") ?? .AppSettingsTitle
+        let suffix = variables.getString("settings-title-punctuation") ?? ""
+
+        navigationItem.title = "\(title)\(suffix)"
```

Comment:
```suggestion
        navigationItem.title = "\(title) \(suffix)"
```
Should we add a space for when we get a suffix or a dash Ex `\(title) - \(suffix)`

<img width="242" alt="image" src="https://user-images.githubusercontent.com/8919439/119730677-0112e080-be44-11eb-89e0-5851a7673415.png">

Reply:
No. The suffix will almost always be the empty string.